low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -65,42 +68,42 @@ class ManifestSupportBundle(object):
 
 
     def make_manifest_support_bundle_tarfile(self, parser, tmpFile):
+        localTempPath = f"{ManifestData}/support_bundle/"
         configure_path = parser[2]
         if configure_path:
             if os.path.exists(configure_path):
-                self.localTempPath = configure_path+"manifest/"
+                localTempPath = configure_path+"manifest/"
             else:
                 msg = "Given path %s doesn't exist" %(configure_path)
                 tmpFile.close()
                 os.remove(onGoingRequestPath)
                 raise Exception(msg)
```

Comment:
Write one common exception class for the manifest file as we did for sspl support bundle. and cleanup all temp files from that exception class. means any error occurs ten it will cleanup temp data.

Reply:
Done